sandbox/templates/richie/base.html

```diff
@@ -12,12 +12,95 @@
+                    {% show_menu 0 100 100 100 %}
+                </ul>
+                <ul class="topbar__menu__list topbar__menu__list--aside">
+                    <li class="topbar__menu__list__item topbar__menu__list__item--signup">
+                        <a class="topbar__menu__list__item__link" href="#">Sign up</a>
+                    </li>
+                    <li class="topbar__menu__list__item topbar__menu__list__item--login">
+                        <a class="topbar__menu__list__item__link" href="#">Login</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        {% endblock body_header %}
 
-            {% endblock content %}
+        <div class="body-content">
+        {% block content %}{% endblock content %}
         </div>
+
+        {% block body_footer %}
+        <div class="body-footer">
+            <div class="body-footer__menu">
+                <ul class="body-footer__menu__list">
+                    <li class="body-footer__menu__list__item">
+                        <a href="#" class="body-footer__menu__list__item__link">About</a>
+                    </li>
+                    <li class="body-footer__menu__list__item">
+                        <a href="#" class="body-footer__menu__list__item__link">Lorem ipsum</a>
+                    </li>
+                    <li class="body-footer__menu__list__item">
+                        <a href="#" class="body-footer__menu__list__item__link">Machin</a>
+                    </li>
+                    <li class="body-footer__menu__list__item">
+                        <a href="#" class="body-footer__menu__list__item__link">Bidule en truc</a>
+                    </li>
+                    <li class="body-footer__menu__list__item">
+                        <a href="#" class="body-footer__menu__list__item__link">Etc..</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="body-footer__brand">
+                <a href="/" class="body-footer__brand__link">
+                    <img src="{% static "images/logo.png" %}" class="body-footer__brand__logo" alt="">
+                </a>
+            </div>
+            <div class="body-footer__mentions">
+                <p>{% now "Y" %} &copy; Acme {% blocktrans %}All Rights Reserved.{% endblocktrans %}</p>
+            </div>
+            <div class="body-footer__poweredby">
+                <a href="https://github.com/openfun/richie" class="body-footer__poweredby__link">
+                    <small>Powered by</small>
+                    <strong>Richie</strong>
+                </a>
+            </div>
+        </div>
+        {% endblock body_footer %}
+
         {% render_block "js" %}
+        <script type="text/javascript">
+        document.addEventListener('DOMContentLoaded', () => {
+            // Get all topbar burger elements
+            const $topbarBurgers = Array.prototype.slice.call(document.querySelectorAll('.topbar__hamburger'), 0);
```

Comment:
What's the `$` prefix' meaning?

Reply:
It's a common practice when your object is a DOM element to quickly see it.